test/common/util.ts

```diff
@@ -51,6 +51,9 @@ export function $x(
  */
 export var selectors = {
     nav: "#stage > #nav",
+    modal: "#stage",
     tab: "//hx-tablist/hx-tab",
     visreg: "*[data-visreg]",
-}
+    openModal: basePath+"/hx-disclosure",
+    closeModal: basePath+"//hx-modal/div/button",
+}
```

Comment:
Don't worry about creating a short hand for this. Just include it in both entries.

Reply:
Thank you Andrew. Removed the shorthand.